src/help/axiosUtils.ts

```diff
@@ -0,0 +1,15 @@
+import type {AxiosRequestConfig} from 'axios';
+
+export const axiosRequestHeaders = (apikey: string): AxiosRequestConfig => {
+  const authorizationHeader = {
+    Authorization: `Bearer ${apikey}`,
+  };
+
+  const documentsRequestHeaders = {
+    ...authorizationHeader,
+    'Content-Type': 'application/json',
+    Accept: 'application/json',
+  };
```

Comment:
Ain't the double object + spread a bit overkill?

Reply:
Yes, but it's because it will be reused multiple times later by both the stream and push strategies